### DISCUSSION TITLE
Update stack to temporarily shutdown servers requires optional fields to be filled out

### DISCUSSION THREAD

**Discussion by mikemainguy** [mozilla/hubs#5657] [Hubs Cloud Questions]
When I follow the instructions to temporarily shutdown my stack, the config asks for optional fields to be mandatory.  As I don't know which ones I can/should just put random things into, I'm hesitant to fill anything out.  redirect url and ssh CIDR I understand, but the backup restore and route53 cert stuff has me confused.
![Screen Shot 2022-08-14 at 07 57 28](https://user-images.githubusercontent.com/554154/184538444-fe080e6e-5187-481f-8596-13add26365af.png)
![Screen Shot 2022-08-14 at 07 57 43](https://user-images.githubusercontent.com/554154/184538450-a2543819-4934-46ac-a1d0-ed4cb1215d78.png)
![Screen Shot 2022-08-14 at 07 57 53](https://user-images.githubusercontent.com/554154/184538457-0ebf491a-c925-4cb4-82a5-1fc5fcceb003.png)
Any advice on what can/should/is safe to enter in these fields?

**Comment by pattersonbl2** [comment]
You wouldn't need to back up and restore when taking your instance offline.  There should be a simple parameter change to take your stack offline if you are doing a backup and restoring that more complex than you need. 
<img width="1355" alt="Screen Shot 2022-08-15 at 11 47 12 AM" src="https://user-images.githubusercontent.com/28871018/184668908-13fe1085-d466-49bd-b032-512a47530156.png">
